src/services/stringCompletions.ts

```diff
@@ -439,7 +439,7 @@ function getStringLiteralCompletionEntries(sourceFile: SourceFile, node: StringL
             const literals = contextualTypes.types.filter(literal => !tracker.hasValue(literal.value));
             return { kind: StringLiteralCompletionKind.Types, types: literals, isNewIdentifier: false };
         default:
-            return fromContextualType();
+            return fromContextualType() || fromContextualType(ContextFlags.None);

```

Comment:
a more conservative approach would be to do this:
```ts
        case SyntaxKind.ArrayLiteralExpression:
            return fromContextualType() || fromContextualType(ContextFlags.None);
        default:
            return fromContextualType();
```

i went for a broad fix~ but perhaps some node types should not be subjects for those retries?